quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/cli/PicocliTest.java

```diff
@@ -88,6 +91,20 @@ public void start() {
             // skip
         }
 
+        @Override
+        protected void initProfile(List<String> cliArgs, String currentCommandName) {
+            super.initProfile(cliArgs, currentCommandName);
+            config = createConfig();
+        }
+
+        @Override
+        public void validateConfig(List<String> cliArgs, AbstractCommand abstractCommand) {
+            if (abstractCommand.getName().equals(Build.NAME)) {
+                //PersistedConfigSource.getInstance().getConfigValueProperties().clear();
+            }
+            super.validateConfig(cliArgs, abstractCommand);
+        }
```

Comment:
What do you want to do here?

Reply:
That's no longer necessary, I had first placed some changes there but then realized the Build command should not be running validation with the persisted config source - that means it will use any persisted values effectively as defaults.